CIP-????/README.md

```diff
@@ -0,0 +1,119 @@
+
+The content of the payload will be included in the protected header of the COSESign1 signature, hence the content effects directly the behavior and security of the system. The payload MUST have the following fields: 
+
+1. The field ```url``` MUST contain the full path to the endpoint, where the payload will be processed.
+
+2. Sometimes, the endpoint ```url``` field is not enough to determine its purpose. The user should understand perfectly the objective of the payload which he or she is signing. That's why the payload MUST contain an ```action``` field with a descriptive text containing the purpose of the payload. 
+
+3. The payload MUST include a UNIX timestamp. The ```timestamp``` field is used as a nonce and a mark for the payload expiration. This field is very imporant in case the payload is compromised.
+
+Additional fields MAY be included in the payload. Depending on the process, it could be useful to include some aditional fields in the protected header of the signature. For example, the email information should be included in the protected header in a registration request. In that way, this payload can be used only to register that specific email and it can not be tampered. 
+
+```JSON
+{
+    "url": "http://example.com/signup",
+    "action": "Sign up",
+    "timestamp": 1673261248,
+    ["email": "email@example.com"] <- Optional
+}
+```
+
+### Wallet specification
+
+The wallets can improve the overall security implementing the following guidelines. We RECOMMEND to show in a structured way the payload information for shake of clarity. This information should be well understanded by the users before the payload is signed.  
+
+The ```url``` field provides information about the hostname of the application. This hostname MUST be included in the wallet allow list. If a known domain A tries to sign a payload for an unknown domain B, you will be prompted with permission popup making more obvious the cross-domain interaction. When possible, the wallet SHOULD warn the user if a payload is for a different domain.
+
+The wallet SHOULD update the ```timestamp``` field to the current time just before the signature. This field ideally should match the moment just before the signature, thereby the server recieves the more updated payload possible. 
+
+### Server processing
+
+The server has ultimate responsibility of processing correctly the requests. We use the content to validate the payload. The request will be processed with the following steps:
+
+1. The server MUST check the action and the endpoint included in the request. Each route to an endpoint MUST have an associated action and a URL. The first step is to check that they match with the parameterized action.
```

Comment:
What request are we talking about here and, how does the server "check the action"? What is being sent to the server, the wallet etc..? 

Perhaps a little diagram (even a plain ASCII one) could help better show the interaction between the actors.

Reply:
@KtorZ I've already created an ASCII diagram. I hope it clarifies the process.

Reply:
@jmagan I can see you've added at the beginning of this section (https://github.com/jmagan/CIPs/blob/master/CIP-%3F%3F%3F%3F/README.md#specification) so @KtorZ marking this as resolved.